modules/ROOT/pages/managing-your-account.adoc

```diff
@@ -180,12 +180,21 @@ You can modify your contact information, change your password, and see a list of
 
 == Reset Your Password
 
+When you reset your password, your new password must be between 8 and 64 characters in length with at least one uppercase letter, one lowercase letter, and one number. Your password can contain only the following: 
```

Comment:
i'm not sure we need this
```
Your password can contain only the following: 
** Upper and lower case characters
** Numbers
** Non-alphanumeric symbols
```
was there a reason why you added that in?

Reply:
can we replace 
```
Your password can contain only the following: 
** Upper and lower case characters
** Numbers
** Non-alphanumeric symbols
```

with

```
Your password can contain ASCII-standard characters only, and accents and accented characters aren't supported.
```

Reply:
```suggestion
When you reset your password, your new password must be between 8 and 64 characters in length with at least one uppercase letter, one lowercase letter, and one number. Your password must contain only ASCII-standard characters.